modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -10,19 +10,19 @@ module ClaimsApi
   class EwsUpdater
     include Sidekiq::Worker
     FILE_5103 = 'Y'
+    OMITTED_FIELDS = %w[contentions dvlpmt_items letters name status_messages station_profile stn_suspns_prfil].freeze
 
     def perform(ews_id)
       ews = ClaimsApi::EvidenceWaiverSubmission.find(ews_id)
       bgs_claim = benefit_claim_service(ews).find_bnft_claim(claim_id: ews.claim_id)
 
-      if bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
-        ews.status = ClaimsApi::EvidenceWaiverSubmission::UPDATED
-      else
+      unless bgs_claim&.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
```

Comment:
moved this around so we only set "updated" when it's entirely complete. Granted it was still technically doing that because of the `ews.save` being below the failing call.